crates/bevy_ui/src/accessibility.rs

```diff
@@ -1,22 +1,18 @@
-use crate::{
-    experimental::UiChildren,
-    prelude::{Button, Label},
-    widget::{ImageNode, TextUiReader},
-    ComputedNode,
-};
+use crate::TextUiReader;
+use crate::{experimental::UiChildren, prelude::Button, widget::ImageNode};
+use crate::{prelude::Label, ComputedNode};
 use bevy_a11y::AccessibilityNode;
-use bevy_app::{App, Plugin, PostUpdate};
+use bevy_app::{App, Plugin};
 use bevy_ecs::{
-    prelude::{DetectChanges, Entity},
+    prelude::Entity,
     query::{Changed, Without},
-    schedule::IntoSystemConfigs,
     system::{Commands, Query},
-    world::Ref,
 };
 use bevy_render::{camera::CameraUpdateSystem, prelude::Camera};
 use bevy_transform::prelude::GlobalTransform;
 
-use accesskit::{Node, Rect, Role};
+use accesskit::Rect;
+use accesskit::{Node, Role};
```

Comment:
These changes don't make a lot of sense to me. Items are removed here but then referenced with their FQN further down (e.g., `Ref`).

Reply:
I can change the style. While working on it, I felt like it was often easier to avoid feature-gating an import by using the FQN.

Reply:
That is fair, perhaps I missed a feature gate that was further up.